imap/caldav_alarm.c

```diff
@@ -1605,14 +1605,15 @@ static int process_futurerelease(struct caldav_alarm_data *data,
             /* Get the response code and error text.
                We treat anything other than 5xx as a temp failure */
             code = smtpclient_get_resp_code(sm);
+            if (code >= 500) {
```

Comment:
Is there a reason that this check was moved from below?

Reply:
the conditional `if (code)` was redundant, as it is always true if `if (code >= 500)`